### PR TITLE
Update on-boarding.rst

### PR DESCRIPTION
**Pull request by wiersgallak**

### CODE REVIEW

File: source/deployment/on-boarding.rst [hunk 1]
```diff
@@ -9,6 +9,11 @@ DO NOT manipulate the Mattermost database
   - In particular, DO NOT manually delete data from the database directly. Mattermost is designed as a continuous archive and cannot be supported after manual manipulation.
   - If you need to permanently delete a team or user, use the `Command Line Tool <../administration/command-line-tools.html>`__.
 
+Updating the Support Email & Help Link
+---------------------------------------
+
+Update the Support Email and Help Link in Mattermost to provide your end users a resource for password resets or questions on their Mattermost account.  The Support Email is used on email notifications and during tutorial for end users to ask support questions.  The Help Link is on the Mattermost login page, sign-up pages, and Main Menu. Many organizations use this URL to link to their help desk ticketing system.
```
Comment (jasonblais) [inline source/deployment/on-boarding.rst]: where are they updated?

Comment (wiersgallak) [reply source/deployment/on-boarding.rst]: @jasonblais - thanks for catching, totally forgot to add that!

File: source/deployment/on-boarding.rst [hunk 2]
```diff
@@ -9,6 +9,11 @@ DO NOT manipulate the Mattermost database
   - In particular, DO NOT manually delete data from the database directly. Mattermost is designed as a continuous archive and cannot be supported after manual manipulation.
   - If you need to permanently delete a team or user, use the `Command Line Tool <../administration/command-line-tools.html>`__.
 
+Updating the Support Email & Help Link
+---------------------------------------
+
+Update the Support Email and Help Link in Mattermost under **System Console > Site Configuration > Customization** to provide your end users a resource for password resets or questions on their Mattermost account.  The Support Email is used on email notifications and during tutorial for end users to ask support questions.  The Help Link is on the Mattermost login page, sign-up pages, and Main Menu. Many organizations use this URL to link to their help desk ticketing system.
```
Comment (jasonblais) [inline source/deployment/on-boarding.rst]:
```suggestion
Update the Support Email and Help Link in Mattermost under **System Console > Site Configuration > Customization** to provide your users a resource for password resets or questions on their Mattermost account.

 - The Support Email is used on email notifications and during tutorial for users to ask support questions.
 - The Help Link is on the Mattermost login page, sign-up pages, and Main Menu. Many organizations use this URL to link to their help desk ticketing system. 
```

Comment (jasonblais) [reply source/deployment/on-boarding.rst]: Propose breaking the two settings to bullets, and referring to `end users` as just `users`

**Pull request merged**